### PR TITLE
Update Script Editor to be Unit, Lesson Group and Lesson

### PR DESCRIPTION
**Pull request by dmcavoy**
Moves to having the top level card be Unit, then Lesson Groups, and Lessons are now the draggable token on the script edit page. 

Note: The Script edit gui is still broken here (aka it can't save). This just gets us to the right UI. 

Screen Shot 2020-09-29 at 12 08 20 AM

## Links

- [PLAT-268](https://codedotorg.atlassian.net/browse/PLAT-268)
- [PLAT-262](https://codedotorg.atlassian.net/browse/PLAT-262)

## Testing story

Updated tests to match the new updated components.

# Reviewer Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

**Comment by davidsbailey** [comment]
@dmcavoy it looks like there are quite a few files being added and removed. Are any of the new files moved/renamed copies of the deleted files? 

Also, from the screenshot, it seems like we might want to take "Levels" out of the "Lessons and Levels" heading, but I'm fine with keeping it if that's intentional.

**Comment by dmcavoy** [comment]
> @dmcavoy it looks like there are quite a few files being added and removed. Are any of the new files moved/renamed copies of the deleted files?

@davidsbailey - Yes you are right there are moved and renamed files. I actually redid this whole PR to explicitly `git mv` everything and it didn't seem to work. Let me giving you a mapping to hopefully help

| Object | Old File | New File|
| -- | -- | --|
| Container for all the Blue Cards | LessonGroups.jsx | None - no longer needed since only one blue card|
| Blue Cards | LessonGroupCard.jsx | UnitCard.jsx|
| White Card| LessonCard.jsx | LessonGroupCard.jsx|
| Token | None | LessonToken.jsx|


NewLessonGroupInput.jsx - Is no longer needed. It was the way we were adding new lesson groups but I found a better way to do it.

> Also, from the screenshot, it seems like we might want to take "Levels" out of the "Lessons and Levels" heading, but I'm fine with keeping it if that's intentional.

Thats a good catch. I can update that.

**Comment by dmcavoy** [comment]
Screen Shot 2020-10-02 at 1 01 20 PM

**Comment by davidsbailey** [comment]
Thanks Dani! 👀   looking...

File: apps/test/unit/lib/levelbuilder/script-editor/UnitCardTest.jsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+          position: 2
+        }
+      ]
+    }
+  ]
+};
+
+describe('UnitCard', () => {
+  it('displays UnitCard correctly', () => {
+    let wrapper = shallow(<UnitCard {...defaultProps} />);
```
Comment (davidsbailey) [inline apps/test/unit/lib/levelbuilder/script-editor/UnitCardTest.jsx:54]: before January launch, this component should have a test which uses `mount` to render LessonGroup and Lesson cards within it.

**Review by davidsbailey — APPROVED** [review]
Looks good! I see two areas for future cleanup here:
* PLAT-23 fix dragging bugs by moving from react-motion to react-draggable-list
* PLAT-374 move Script Editor off of position-based logic

File: apps/test/unit/lib/levelbuilder/script-editor/UnitCardTest.jsx (hunk 1 above)
Comment (dmcavoy) [reply apps/test/unit/lib/levelbuilder/script-editor/UnitCardTest.jsx:54]: Tracked: https://codedotorg.atlassian.net/browse/PLAT-375

**Pull request merged**